pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+
+[tool.black]
+line-length = 100
+target-version = ['py36', 'py37', 'py38']
```

Comment:
Any particular reason why `py39` is not (yet) included?

Reply:
Thanks, probably ok to add. See here: 6976ad4